src/styles/index.scss

```diff
@@ -28,6 +28,12 @@
   }
 }
 
+.image-card-group .bx--no-gutter-sm:nth-of-type(even) {
+  .bx--image-card--dark {
+    border-left: 1px solid #e0e0e0;
```

Comment:
do we have a color token that we can use instead?

Reply:
I've updated it to use a color token instead, Thanks!